cmd/osde2e/cleanup/cmd.go

```diff
@@ -255,9 +274,10 @@ func run(cmd *cobra.Command, argv []string) error {
 		errorMessage := `"full":"` + buildFile + `",`
 		s3ErrorMessage := `"s3":" S3 Errors: \n ` + s3ErrorBuilder.String() + `",`
 		iamErrorMessage := `"iam":" IAM Errors: \n` + iamErrorBuilder.String() + `",`
-		ipErrorMessage := `"ip":" ElasticIP Errors: \n` + ipErrorBuilder.String() + `"}`
+		ipErrorMessage := `"ip":" ElasticIP Errors: \n` + ipErrorBuilder.String() + `",`
+		ec2ErrorMessage := `"ec2":" EC2 Errors: \n` + ec2ErrorBuilder.String() + `"}`
```

Comment:
Another: while things are like this currently, it is fine and we can proceed. I don't recommend doing something like this on a the future project though, this is super fragile. Just create a struct and json encode it.